common/src/main/java/org/apache/celeborn/common/protocol/TransportModuleConstants.java

```diff
@@ -21,6 +21,17 @@ public class TransportModuleConstants {
   public static final String PUSH_MODULE = "push";
   public static final String REPLICATE_MODULE = "replicate";
   public static final String FETCH_MODULE = "fetch";
-  public static final String RPC_MODULE = "rpc";
+
+  // RPC module used by the application components to communicate with each other
+  // This is used only at the application side.
+  public static final String RPC_APP_MODULE = "rpc_app";
+  // RPC module used to communicate with/between server components
+  // This is used both at server (master/worker) and application side.
+  public static final String RPC_SERVER_MODULE = "rpc_server";
```

Comment:
maybe "app:service" and "client:server", we used the term pair "client:server" in some places, for example, the master client could be either on the app or worker.

Reply:
Doesn't this not need to indicate it is specifically for rpc ? and not for other transport modules ?

Reply:
sorry for misleading, I mean rpc_app and rpc_service

Reply:
Ah, service makes sense - agree.
Will rename

Reply:
Done, PTAL